client/src/pages/reports/Form.js

```diff
@@ -305,8 +305,53 @@ const ReportForm = ({
           currentUser.position && currentUser.position.organization
         const locationFilters = {
           activeLocations: {
-            label: "Active locations",
+            label: "All",
             queryVars: { status: Model.STATUS.ACTIVE }
+          },
+          geographical: {
+            label: Location.humanNameOfType(
+              Location.LOCATION_TYPES.GEOGRAPHICAL_AREA
+            ),
+            queryVars: {
+              status: Model.STATUS.ACTIVE,
+              type: Location.LOCATION_TYPES.GEOGRAPHICAL_AREA
+            }
+          },
+          pinpoint: {
+            label: Location.humanNameOfType(
+              Location.LOCATION_TYPES.PINPOINT_LOCATION
+            ),
+            queryVars: {
+              status: Model.STATUS.ACTIVE,
+              type: Location.LOCATION_TYPES.PINPOINT_LOCATION
+            }
+          },
```

Comment:
Users can't choose `PINPOINT_LOCATION` (it's a parent type for advisor or principal location).

Reply:
We've decided to allow this for now. At some point, we may introduce a new sub-type of pinpoint location (e.g. PPU or PPO), but not right now.